ops/dev/test-pacs-contract.ps1

```
@@ -0,0 +1,164 @@
+      3. Three indexes exist
+      4. sp_TerraFusion_HealthCheck exists and runs
+      5. Property read (by ID and GeoID)
+      6. Ownership + Assessment History queries
+    Exit 0 = all checks pass. Exit 1 = failure.
+.PARAMETER SaPassword
+    SQL Server SA password. Default: TF_Pacs2026!
+.PARAMETER ContainerName
+    Docker container name. Default: tf-mssql
+#>
+param(
+    [string]$SaPassword = "TF_Pacs2026!",
```

PSScriptAnalyzer (1.24.0)

[warning] Parameter '$SaPassword' should not use String type but either SecureString or PSCredential, otherwise it increases the chance to expose this sensitive information.

Suggested fix: Set $SaPassword type to SecureString; Set $SaPassword type to PSCredential

(PSAvoidUsingPlainTextForPassword)